wallet-new/docs/spec.tex

```diff
@@ -3009,38 +3025,533 @@ \subsection{Use cases}
+exactly 10k and all payments exactly 1k.
+We can see the large value being deposited, and then shifting to the left in the
+histogram as it is getting used for deposits, each time decreasing that large
+output by 1k. Indeed, this takes 10 slots on average, which makes sense given
+the 10:1 ratio; moreover, the \emph{average value} of the ``large output'' in such a
+10-slot cycle is 5k, explaining why we are getting 5.0 change:payment ratio.
+
+The algorithm however is \emph{not} creating dust outputs; the 1k change outputs
+it is generating \emph{are} getting used, and the size of the UTxO is perfectly
+stable. Indeed, back in Figure~\ref{fig:inputselection:normal_1to10_randomOn} we
+can see that the size of the UTxO tracks the balance perfectly; moreover, the
+vast majority of transactions only use a single input, which is what we'd expect
+for a 10:0 deposit:payment ratio.
+
+\subsubsection{Real data}
+
+\paragraph{\texttt{MoneyPot.com}}
+
+Ideally, of course, we run the simulation against \emph{real} event streams from
+existing wallets. Unfortunately, such data is hard to come by. Erhardt was able
+to find one such dataset, provided by \texttt{MoneyPot.com}
+\citep{MoneyPotdataset}. Figure~\ref{fig:inputselection:MoneyPot_randomOn} shows
+the results of running our algorithm on this dataset.
+
+\begin{figure}[p]
+\begin{center}
+\scriptsize
+\begin{tabular}{ll}
+\input{spec/MoneyPot-randomOn.tex} &
+\input{spec/MoneyPot-randomOn-distr.tex}
+\end{tabular}
+\end{center}
+\caption{\label{fig:inputselection:MoneyPot_randomOn}
+  \texttt{Random-Improve}, using the MoneyPot data set. There is a rougly 2:1
+  deposit:payment ratio. Values have been scaled. Log scale on the x-axis.
+}
+\end{figure}
+
+A few observations are in order here. First, there are quite a few deposits and
+payments close to 0, just like in an exponential distribution. Moreover,
+although we have many values close to 0, we also have some \emph{huge} outliers; the
+payments are closely clustered together, but there is a $10^9$ difference between
+the smallest and largest deposit, and moreover a $10^5$ difference between the
+largest deposit and the largest payment. It is therefore not surprising that we
+end up with a relatively large change:payment ratio. Nonetheless, the algorithm
+is behaving well, with the size of the UTxO tracking the balance nicely, with an
+average UTxO size of 130 entries. The average number of outputs is 3.03, with
+50\% of transactions using just one input, and 90\% using 6 or fewer.
+
+\paragraph{Cardano Exchange}
+
+One of the large Cardano exchange node has also helped us with
+some anonymised data (deposits and payments), similar in nature to the
+MoneyPot dataset albeit significantly larger. Coming from an exchange node,
+however, this dataset is \emph{very much} skewed towards deposits, with a
+deposit:payment ratio of roughly 30:1. Under these circumstances, of course,
+coin selection alone cannot keep the UTxO size small
```

Comment:
The "of course" is not really obvious. It doesn't, that's true, but we should simply admit that this illustrates a limit. It's ok for 10:1 but 30:1 is apparently stretching things too far. We can also point out that executing additional maintenance transactions in this case would be straightforward and should deal with the problem.